easybuild/easyconfigs/g/GULP/GULP-6.1-foss-2021b.eb

```diff
@@ -0,0 +1,60 @@
+# Contribution from Imperial College London / UK 
+# Based on GULP-6.0-foss-2020b.eb 
+# uploaded by J. Sassmannshausen
+
+easyblock = 'MakeCp'
+
+name = 'GULP'
+version = '6.1'
+
+homepage = 'https://gulp.curtin.edu.au/gulp/'
+description = """GULP is a program for performing a variety of types of simulation on materials
+ using boundary conditions of 0-D (molecules and clusters), 1-D (polymers), 2-D (surfaces, slabs
+ and grain boundaries), or 3-D (periodic solids)Band Unfolding code for Plane-wave based calculations"""
+
+toolchain = {'name': 'foss', 'version': '2021b'}
+toolchainopts = {'usempi': True}
+
+# download requires registration, https://gulp.curtin.edu.au/gulp/request.cfm?rel=download
```

Comment:
can you use `download_instructions` so that EasyBuild gives a more helpful message if the sources aren't found?

Reply:
Done. I should have remembered it. I hope that is better now. Thanks!